src/itinerary/index.unit.tsx

```diff
@@ -123,4 +123,56 @@ describe('Itinerary component', () => {
     const itinerary = renderer.create(<Itinerary places={places} />).toJSON()
     expect(itinerary).toMatchSnapshot()
   })
+
+  it('Should use subLabel in key if it\'s a string', () => {
```

Comment:
nit: you can use double quotes to avoid escaping single quote

`"Should use subLabel in key if it's a string"`